layers/meta-tegra-support/recipes-ota/tegra-tools/tegra-sysinstall-tools_git.bb

```diff
@@ -0,0 +1,27 @@
+DESCRIPTION = "System installation scripts"
+HOMEPAGE = "https://github.com/madisongh/tegra-sysinstall"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=5e5799d70d07956d01af05a7a92ea0d7"
+
+DEPENDS = "zlib"
+
+SRC_REPO ?= "github.com/madisongh/tegra-sysinstall.git;protocol=https"
+SRCBRANCH ?= "master"
+SRC_URI = "git://${SRC_REPO};branch=${SRCBRANCH}"
+SRCREV = "${AUTOREV}"
+PV = "1.3.99+git${SRCPV}"
+
+COMPATIBLE_MACHINE = "(tegra)"
+
+S = "${WORKDIR}/git"
+
+inherit autotools
+
+FILES_${PN} += "${datadir}/tegra-sysinstall"
+EXTRA_RDEPENDS = ""
+EXTRA_RDEPENDS_cryptparts = "cryptsetup keystore-tools"
+RDEPENDS_${PN} = "tegra-boot-tools tar ${EXTRA_RDEPENDS} \
+                  bash curl util-linux-blkid util-linux-lsblk util-linux-mountpoint \
+                  parted gptfdisk e2fsprogs util-linux-mkfs util-linux-mount \
+                  util-linux-umount tegra-fuse-tool tegra-eeprom-tool-boardspec \
+                  sysinstall-partition-layout mtd-utils"
```

Comment:
You can drop this recipe altogether.  The versioned one is the right one to include.